src/main/java/com/conveyal/datatools/manager/controllers/DumpController.java

```diff
@@ -207,12 +207,13 @@ public static boolean updateSnapshotMetadata (String jsonString) {
 
     /**
      * Load a v2 JSON dump (i.e., objects with the class structure immediately before the MongoDB migration).
+     * @param req
      */
-    private static boolean loadLegacy(String jsonString) {
+    private static boolean loadLegacy(Request req) {
```

Comment:
There may be cases in the future where it's helpful to be able to just load a JSON string directly into this loadLegacy method (rather than going through a spark request).  Could you keep the bulk of this function's logic and the previous function signature intact (single String param) and maybe just wrap this within the standard `(Request req, Response res)` function signature for the spark request?